src/CompleteBipartiteGraph.jl

```diff
@@ -0,0 +1,158 @@
+
+
+# =======================================================
+#         CompleteBipartiteGraph struct
+# =======================================================
+
+"""
+    CompleteBipartiteGraph <: AbstractGraph
+
+A structure representing an undirected bipartite graph.
+
+
+See also: [`LightGraphs.complete_bipartite_graph`](@ref)
+"""
+struct CompleteBipartiteGraph{T<:Integer} <: LG.AbstractGraph{T}
+    m::T
+    n::T
+
+    function CompleteBipartiteGraph{T}(m, n) where {T}
+       
+        m = convert(T, m)
```

Comment:
shouldn't conversion be in a secondary constructor (not sure it is zero-cost when trivial)

Reply:
Should be zero-cost because we have this definition in Base:
```julia
convert(::Type{T}, x::T) where {T<:Number} = x
```